### PR TITLE
MOTECH-2324: When we use the char datatype, field name is automatically filled by 'char'

### PR DESCRIPTION
**Pull request by shanmukhm**
- The defaultName for the Character type is changed from 'char' to 'character'

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

File: platform/mds/mds/src/main/resources/db/migration/default/V4__MOTECH-1659.sql [hunk 1]
```diff
@@ -20,7 +20,7 @@ LIMIT 1;
 -- Adding Character to Type table
 --
 INSERT INTO "Type"
-SELECT "id" + 1, 'mds.field.description.character','mds.field.character','char','java.lang.Character'
+SELECT "id" + 1, 'mds.field.description.character','mds.field.character','character','java.lang.Character'
```
Comment (sebbrudzinski) [inline platform/mds/mds/src/main/resources/db/migration/default/V4__MOTECH-1659.sql]: You must not edit existing migrations. Create a new one if necessary.

File: platform/mds/mds/src/main/resources/db/migration/mysql/V4__MOTECH-1659.sql [hunk 2]
```diff
@@ -20,7 +20,7 @@ LIMIT 1;
 -- Adding Character to Type table
 --
 INSERT INTO Type
-SELECT id + 1, 'mds.field.description.character','mds.field.character','char','java.lang.Character'
+SELECT id + 1, 'mds.field.description.character','mds.field.character','character','java.lang.Character'
```
Comment (sebbrudzinski) [inline platform/mds/mds/src/main/resources/db/migration/mysql/V4__MOTECH-1659.sql]: You must not edit existing migrations. Create a new one if necessary.

**Comment by shanmukhm** [comment]
Okay I'll create a new migration for these changes.

**Comment by shanmukhm** [comment]
This branch is messed up somehow. I'll create a new PR for this ticket.

**Pull request closed without merging**